mysql scripts/table_creation/appointmentDB.sql

```diff
@@ -0,0 +1,74 @@
+-- VITA Project
+-- UNL CSE Ambassadors 2017
+
+USE vita;
+
+DROP TABLE IF EXISTS AppointmentQuestionAnswer;
+DROP TABLE IF EXISTS Appointment;
+DROP TABLE IF EXISTS Location;
+DROP TABLE IF EXISTS UserAnswer;
+DROP TABLE IF EXISTS PossibleAnswer;
+DROP TABLE IF EXISTS Question;
+DROP TABLE IF EXISTS QuestionAccessories;
+
+CREATE TABLE QuestionAccessories (
```

Comment:
The field "required" is in Question table. I'll defer to @CollinsSpencer if the spellcheck, readonly, disabled, etc. fields are necessary -- I don't want to put in useless columns if there's no practical client-side use for them. Validation is outsourced to @CollinsSpencer JQuery stuff.